content/english/arduino/4-appendix/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Appendix"
+description: ""
+draft: true
+weight: 1
+translationKey: "arduino-4"
+---
+
+# Appendix
+
+## The components, in detail
+
+Imagine our circuit is a play. Before the final performance in front of an audience, we need a place to practice. This is where the breadboard comes in. 
+> > Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together. 

```

Comment:
I'm not a huge fan of using the quote indent for this. Can we just use a bulleted list instead?